tests/schema/models.py

```diff
@@ -147,6 +147,17 @@ class Meta:
         apps = new_apps
 
 
+class BookWithIndex(models.Model):
+    author = models.ForeignKey(Author, models.CASCADE)
+    title = models.CharField(max_length=100)
+
+    class Meta:
+        apps = new_apps
+        indexes = [
+            models.Index(fields=['author', 'title'], name='index_with_fk')
+        ]
+
```

Comment:
`BookWithIndex` is used only in a single test, so I would rather create an inline model and use `@isolate_apps()`.